target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -73,6 +95,18 @@ define Device/ubnt_nano-m
 endef
 TARGET_DEVICES += ubnt_nano-m
 
+define Device/ubnt_nanostationacl
+  $(Device/ubnt-wa)
+  DEVICE_TITLE := Ubiquiti Nanostation AC loco
+  DEVICE_PACKAGES += kmod-ath10k ath10k-firmware-qca988x
+  IMAGE_SIZE := 15744k
+  IMAGES := factory.bin sysupgrade.bin
+  IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size $$$$(IMAGE_SIZE) | append-metadata
```

Comment:
You can drop this as well. It matches more or less what is defined in the [ath79 toplevel Makefile](https://github.com/openwrt/openwrt/blob/aed46928fa5a0cd9df076cb4d11c76a8413cdae4/target/linux/ath79/image/Makefile#L65).

Reply:
Again, thanks for pointing that out. I've copied over a lot of stuff from my initial ar71xx support commit from January without giving it a second thought